memstore/src/test.rs

```diff
@@ -585,11 +584,19 @@ where
         Ok(())
     }
 
-    pub async fn first_id_in_log(builder: &B) -> anyhow::Result<()> {
```

Comment:
The test about `first_id_in_log()` and `last_id_in_log()` should be kept because there are default implementation for them now.

Reply:
done